liquibase-core/src/main/java/liquibase/changelog/annotations/reader/ChangeEntry.java

```diff
@@ -0,0 +1,82 @@
+package liquibase.changelog.annotations.reader;
+
+import java.util.Date;
+import java.util.Objects;
+
+
+public class ChangeEntry {
```

Comment:
My plan here is to make the database provider responsible for storing this object.